sdk/webpubsub/azure-messaging-webpubsub-client/src/test/java/com/azure/messaging/webpubsub/client/GroupMessageTests.java

```diff
@@ -190,8 +198,33 @@ public void testSendMessagePerformance() throws InterruptedException {
         }
     }
 
-    private static class JsonModel {
+    private static class JsonModel implements JsonSerializable<JsonModel> {
```

Comment:
```suggestion
    public static class JsonModel implements JsonSerializable<JsonModel> {
```

`java.lang.IllegalStateException: java.lang.IllegalAccessException` is private